### ISSUE TITLE
fetchFirstColumn - not found when searching 3.0 docs

### ISSUE THREAD

**Issue by jacekkarczmarczyk** [doctrine/doctrine-website#404]
![image](https://user-images.githubusercontent.com/15625235/119344096-86a55d80-bc97-11eb-8c97-02bf42d52b3d.png)

**Comment by greg0ire** [comment]
@jacekkarczmarczyk how is this an issue with `doctrine-website`? `fetchFirstColumn` does not appear under `docs`, does it?

**Comment by jacekkarczmarczyk** [comment]
Not sure if understand, I assument that the site I've linked is a Doctrine website and the name of this repo suggest that it's the source of this site. So if the `fetchFirstColumn` is a method that Doctrine DBAL provides then I'd expect I could find some information on that on the Doctrine website, am I right?
And since I can't find any information on this method then I'm pretty sure that it's the issue with Doctrine website that is worth to submit

**Comment by greg0ire** [comment]
> Not sure if understand, I assument that the site I've linked is a Doctrine website and the name of this repo suggest that it's the source of this site.

Not really, this repo actually just builds from every `docs` directory it finds on every Doctrine repository.

> So if the fetchFirstColumn is a method that Doctrine DBAL provides then I'd expect I could find some information on that on the Doctrine website, am I right?

Correct, but that should be reported to the dbal repo instead. I'm transferring your issue.
